### PR TITLE
ROC-3377: Update CCJ PDF

### PR DESCRIPTION
**Pull request by varma02**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-3377

### Change description ###
pdf changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[* ] No
```

### CODE REVIEW

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 1]
```diff
@@ -7,12 +7,32 @@
...
     }
 
+    /*@page :first {*/
+      /*@top-right {*/
+        /*content: "{{ ccj.claim.referenceNumber }}";*/
+        /*!*noinspection CssNoGenericFontName*!*/
+        /*font-family: "Open Sans";*/
+      /*}*/
+    /*}*/
+
```
Comment (varma02) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: I need help here please - I want to get claim number only on second page as a header...I tried as shown above but doesn't work

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 2]
```diff
@@ -35,6 +55,15 @@
       content: counter(section) ". ";
     }
 
+    .no-border td {
```
Comment (varma02) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: I just moved this class, so all classes are in one section

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html (hunk 1 above)
Comment (varma02) [reply src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: i got it to work now

**Comment by varma02** [comment]
[Defendant-000MC378-county-court-judgment-details.pdf](https://github.com/hmcts/cmc-claim-store/files/1838551/Defendant-000MC378-county-court-judgment-details.pdf)
[Defendant-000MC377-county-court-judgment-details.pdf](https://github.com/hmcts/cmc-claim-store/files/1838552/Defendant-000MC377-county-court-judgment-details.pdf)
[Defendant-000MC376-county-court-judgment-details.pdf](https://github.com/hmcts/cmc-claim-store/files/1838553/Defendant-000MC376-county-court-judgment-details.pdf)

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 3]
```diff
@@ -7,9 +7,33 @@
...
       margin: 10%;
 
+      @bottom-center {
+        content: "For further details of the courts www.gov.uk/find-court-tribunal";
+        font-size: 10px;
+      }
+
+      @bottom-right {
+        content: "© Crown copyright Page " counter(page) " of " counter(pages);
+        /*noinspection CssNoGenericFontName*/
```
Comment (sabahirfan) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: please remove commented out code.

Comment (timja-kainos) [reply src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: that's not commented out code, that's for intelliJ to not complain

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 4]
```diff
@@ -7,9 +7,33 @@
...
-        content: "OCON205A Request for judgment by default (07.17)";
-        font-size: 11px;
+        content: "OCON225";
+        /*noinspection CssNoGenericFontName*/
+        font-size: 10px;
+      }
+
+      @top-right {
+        content: "Claim number: {{ ccj.claim.referenceNumber }}";
+        /*noinspection CssNoGenericFontName*/
```
Comment (sabahirfan) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: please remove commented out code.

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html (hunk 3 above)
Comment (sabahirfan) [reply src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: ok

**Comment by varma02** [comment]
@timja-kainos your comments are addressed except 'Crown copyright doesn't fit on one line like it does on the other PDFs' (I couldn't get it to work yet)

**Comment by varma02** [comment]
[Defendant-000MC389-county-court-judgment-details.pdf](https://github.com/hmcts/cmc-claim-store/files/1840812/Defendant-000MC389-county-court-judgment-details.pdf)


I have moved css content to match same with sealedClaim.html hoping i would solve the issue with Crown copyright doesn't fit on one line but still no luck

**Comment by varma02** [comment]
[Defendant-000MC395-county-court-judgment-details.pdf](https://github.com/hmcts/cmc-claim-store/files/1841264/Defendant-000MC395-county-court-judgment-details.pdf)

**Comment by varma02** [comment]
[Defendant-000MC398-county-court-judgment-details.pdf](https://github.com/hmcts/cmc-claim-store/files/1841820/Defendant-000MC398-county-court-judgment-details.pdf)

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 5]
```diff
@@ -3,12 +3,36 @@
...
 
+      @top-right {
+        content: "Claim number: {{ ccj.claim.referenceNumber }}";
+        font-size: 18px;
+        font-family: "Open Sans";
+      }
+
       @bottom-left {
-        content: "OCON205A Request for judgment by default (07.17)";
+        content: "OCON225";
```
Comment (geek-soft) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: Prototype linked to the story has `N225`, is `OCON225` correct?

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 6]
```diff
@@ -80,41 +109,51 @@
 <body>
 
 <header>
-
   <table>
-    <tr>
-      <td colspan="3"><p>In the County Court Business Centre Online Court</p></td>
+    <tr align="right">
+      <td colspan="3"><h3 class="row-spacing">Request for judgment by default</h3></td>
```
Comment (geek-soft) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: Should be `colspan="2"` as there are only two cells below.

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 7]
```diff
@@ -80,41 +109,51 @@
...
     </tr>
     <tr>
-      <td colspan="2"><h2><strong>Claim number: </strong></h2></td>
-      <td><h2 style="font-weight: normal"> {{ ccj.claim.referenceNumber }}</h2></td>
+      <td colspan="3"><p class="row-spacing">In the County Court Business Centre Online Court</p></td>
```
Comment (geek-soft) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: Should be `colspan="2"` as there are only two cells below.

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 8]
```diff
@@ -173,167 +222,175 @@ <h2>Request for judgment by default</h2>
...
-      </tr>
-      <tr>
-        <td>Date interest is claimed from:</td>
-        <td>
-          {{ ccj.amount.interest.fromDate }}
-        </td>
-      </tr>
-      <tr>
-        <td>Interest daily amount:</td>
+        <td><strong>Amount already paid by defendant to pay:</strong></td>
```
Comment (geek-soft) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: Prototype has `Amount already paid by defendant:`. No `to pay`

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 9]
```diff
@@ -173,167 +222,175 @@ <h2>Request for judgment by default</h2>
...
+      <div class="section-heading">
+        <strong>Total amount</strong>
+      </div>
+      <div class="section-body">
+          <table>
         <tr>
+          <td><strong>Claim amount:</strong></td>
           <td>
-            Name:
+            {{ ccj.amount.totalAmount }}
```
Comment (geek-soft) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: That does not look right. A `ccj.amount.totalAmount` is used both in `Claim amount:` and `Total:` rows.

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html (hunk 5 above)
Comment (varma02) [reply src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: Art asked me to update content

**Comment by geek-soft** [comment]
I am working to address review comments and make it ready for Monday.

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 10]
```diff
@@ -108,13 +137,29 @@ <h2>Request for judgment by default</h2>
   <div class="section-body">
     <table>
       <tr>
-        <td>Full Name:</td>
+        <td><strong>Name:</strong></td>
```
Comment (geek-soft) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: I used the same approach as in latest sealed copy of claim. Prototype introduces some new approach with `Company name`, `Organisation name` and `Contact name`. It does not give example for sole traders and individuals.

I will clarify that tomorrow and either keep current format or implement new format when I have more details how to do so.

**Comment by geek-soft** [comment]
Agreed format:

```
1. Individual

Name:

2. Sole trader

Name:
Company name:

3. Company

Company name:
Contact name:

4. Organisation

Organisation name:
Contact name:
```

**Comment by geek-soft** [comment]
Still need to be tested for interest breakdown but getting there.

**Comment by geek-soft** [comment]
Aslo as requested today `Amount already paid by defendant:` have to appear in section `4. Total amount` under `Claim fee:`.

**Comment by geek-soft** [comment]
Changes are ready for review. I just need to double check implementation for interest breakdown is right. Few sample PDF's enclosed to story in JIRA.

**Comment by varma02** [comment]
Changes after demo to Art Atwal,

1. Add sub-total after claim fee in section 4.
2. In section 4, show claim amount, interest to date of judgement and claim fee in 1 section without bottom-border

**Comment by varma02** [comment]
Latest pdf's attached to jira - 
Defendant-999MC010-county-court-judgment-details.pdf
Defendant-999MC011-county-court-judgment-details.pdf

**Comment by geek-soft** [comment]
Looks good, first thing to get merged in the morning.

**Comment by timja-kainos** [comment]
Integration tests passed, merging

**Pull request merged**